### ISSUE TITLE
[Bug]: orange compact some components are not displayed correctly

### ISSUE THREAD

**Issue by AhmedAmineZr** [Orange-OpenSource/ouds-flutter#630]
### Prerequisites

- [x] I have [searched the backlog](https://github.com/Orange-OpenSource/ouds-flutter/issues) for duplicate or closed feature requests

### Your test device

all

### OS Version

iOS,Android

### App version

1.1.2

### Describe the issue

In Orange Compact, after reviewing version 1.1.2, some components are not displayed correctly:

- [x] Password input: The view icon size is much too small
- [x] Filter chip: The label should be in moderate and not in strong
- [x] Suggestion chip: The label should be in moderate and not in strong
- [x] PIN code input: In the test resolution, the component has a 370px width. I notice an offset of the first input. Moreover, the height should be smaller than what is currently developed.
- [x] Text input: The trailing action is much too small

### Expected Behavior

- [x] Password input: The view icon should be displayed at the correct size according to the design specifications
- [x]  Filter chip: The label text should use moderate font weight instead of strong/bold
- [x] Suggestion chip: The label text should use moderate font weight instead of strong/bold
- [x] PIN code input: At 370px width, all input fields should be properly aligned without any offset. The component height should match the design specifications
- [x] Text input: The trailing action icon/element should be displayed at the correct size according to the design specifications

**Comment by nouha06** [comment]
New TestFlight Alpha upload available 🚀 (iOS)

Please, get the new build from TestFlight app, or contact the project maintainers (@AhmedAmineZr ) to get access.

Display name: Design Toolbox Flutter (660)
Version: 1.2.0
Build type: ALPHA
Build details (GitHub): 630
cc @B3nz01d (product owner)
cc @AhmedAmineZr @nouha06 (dev team)
cc @jerome-regnier (design team)
cc @pya35 (accessibility team)

**Comment by nouha06** [comment]
📣 New Firebase App Distribution Alpha upload available 🚀 (Android)

Please, get the new build from App Tester app, or contact the project maintainers (@AhmedAmineZr ) to get access.

Display name: DesignToolbox (ALPHA) (660)
Version: 1.2.0
Build type: Alpha
Build details (GitHub): 630
cc @B3nz01d (product owner)
cc @AhmedAmineZr @nouha06 (dev team)
cc @jerome-regnier (design team)
cc @pya35 (accessibility team)

**Comment by jerome-regnier** [comment]
Design review Figma : 
https://www.figma.com/design/yKtkEY1DOl7YDvmt7jq0QF/-OUDS-Core-Lib--Review?node-id=71488-47038&t=MxVBCqBvPWCNHJun-1

- Tag : Problème de taille, le dev est trop grand dans sa taille default. En taille small, la taille est bonne mais le texte a un décalage
- Switch : Le composant n’a pas assez de hauteur lorsqu’il n’y a pas la description. En state error, problème de décalage de l’icône alerte, du label, de la description. Le texte description devrait être en rouge
- Tap Bar : Le marker devrait faire 40px
- Checkbox item : Sur le checkbox item sans description, le composant n'a pas la bonne hauteur et il y a un décalage de la checkbox et du label. Avec la description, la hauteur est bonne mais on a toujours le décalage de la checkbox, Label et description.
- Radio buttons item : Décalage du radio, label, extra label et description. En state error, l’extra label et la description devrait être en rouge. Décalage du pictogramme alerte.
- Link text only : Problème de calage du soulignement, principalement sur la version small
- Password input : En state error, problème de size du label sur le cas avec un placeholder
- Phone number input : Le label "Phone number est trop petit lorsqu'on est en focus
- Text input : Avec placehoder, le label est trop petit.

**Comment by AhmedAmineZr** [comment]
like we have discuss with @jerome-regnier, we will make new review with new token when it will be delivered in two weeks.
we will make change only for topography Password and Tag size with icon

cc @nouha06 @paulinea

**Comment by nouha06** [comment]
> Design review Figma :  https://www.figma.com/design/yKtkEY1DOl7YDvmt7jq0QF/-OUDS-Core-Lib--Review?node-id=71488-47038&t=MxVBCqBvPWCNHJun-1
> 
> * Tag : Problème de taille, le dev est trop grand dans sa taille default. En taille small, la taille est bonne mais le texte a un décalage
> * Switch : Le composant n’a pas assez de hauteur lorsqu’il n’y a pas la description. En state error, problème de décalage de l’icône alerte, du label, de la description. Le texte description devrait être en rouge
> * Tap Bar : Le marker devrait faire 40px
> * Checkbox item : Sur le checkbox item sans description, le composant n'a pas la bonne hauteur et il y a un décalage de la checkbox et du label. Avec la description, la hauteur est bonne mais on a toujours le décalage de la checkbox, Label et description.
> * Radio buttons item : Décalage du radio, label, extra label et description. En state error, l’extra label et la description devrait être en rouge. Décalage du pictogramme alerte.
> * Link text only : Problème de calage du soulignement, principalement sur la version small
> * Password input : En state error, problème de size du label sur le cas avec un placeholder
> * Phone number input : Le label "Phone number est trop petit lorsqu'on est en focus
> * Text input : Avec placehoder, le label est trop petit.

Le problème du tag est résolu. Les problèmes concernant les contrôles d'items semblent également corrects. Il est recommandé de tester avec la même version que celle utilisée dans l'application de démonstration.

En ce qui concerne le floating label des champs de texte, il s'agit d'un comportement natif de Flutter. Même si l'on augmente la taille du floating label, Flutter ajuste automatiquement la taille en effectuant un redimensionnement.

**Comment by nouha06** [comment]
📣 New Firebase App Distribution Alpha upload available 🚀 (Android)

Please, get the new build from App Tester app, or contact the project maintainers (@AhmedAmineZr ) to get access.

Display name: DesignToolbox (ALPHA) (685)
Version: 1.3.0
Build type: Alpha
Build details (GitHub): 630
cc @B3nz01d (product owner)
cc @AhmedAmineZr @nouha06 (dev team)
cc @jerome-regnier (design team)
cc @pya35 (accessibility team)

**Comment by nouha06** [comment]
New TestFlight Alpha upload available 🚀 (iOS)

Please, get the new build from TestFlight app, or contact the project maintainers (@AhmedAmineZr ) to get access.

Display name: Design Toolbox Flutter (685)
Version: 1.3.0
Build type: ALPHA
Build details (GitHub): 630
cc @B3nz01d (product owner)
cc @AhmedAmineZr @nouha06 (dev team)
cc @jerome-regnier (design team)
cc @pya35 (accessibility team)

**Comment by nouha06** [comment]
according to ours discuss with @jerome-regnier on review in [figma](https://www.figma.com/design/yKtkEY1DOl7YDvmt7jq0QF?node-id=71488-37215#1709129971) this ticket will be merged
